examples/stm32/nucleo-h743zi-baremetal/Makefile

```diff
@@ -0,0 +1,38 @@
+CFLAGS  ?=  -W -Wall -Wextra -Werror -Wundef -Wshadow -Wdouble-promotion \
+						-Wformat-truncation -fno-common -Wconversion -Wno-sign-conversion \
+						-g3 -Os -ffunction-sections -fdata-sections \
+            -I. -Ih7/Include -Icmsis/CMSIS/Core/Include \
+            -mcpu=cortex-m7 -mthumb -mfloat-abi=soft \
```

Comment:
TODO(): hard FPU

Reply:
Yeah, there is FPU and other TODOs in the mcu.h